src/Xamarin.Android.Build.Tasks/Tasks/GeneratePackageManagerJava.cs

```diff
@@ -216,6 +219,19 @@ void AddEnvironment ()
 						continue;
 					}
 
+					if (lineToWrite.StartsWith ("DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=", StringComparison.Ordinal)) {
+						haveDotnetSystemGlobalizationInvariant = IsEntryValueBooleanTrue (lineToWrite);
+						continue;
+					}
+					if (lineToWrite.StartsWith ("DOTNET_SYSTEM_GLOBALIZATION_USENLS=", StringComparison.Ordinal)) {
+						haveDotnetSystemGlobalizationUseNls = IsEntryValueBooleanTrue (lineToWrite);
+						continue;
+					}
+					if (lineToWrite.StartsWith ("DOTNET_SYSTEM_GLOBALIZATION_APPLOCALICU=", StringComparison.Ordinal)) {
+						haveDotnetSystemGlobalizationAppLocalIcu = IsEntryValueBooleanTrue (lineToWrite);
+						continue;
+					}
+
```

Comment:
Why are you adding this?

Reply:
How otherwise these properties can be passed to MonoVM?

Reply:
They should not be passed in, the code reading them will be removed.

Reply:
ok, I'm putting this PR on hold till we know more